src/jit/flowgraph.cpp

```diff
@@ -25557,38 +25576,60 @@ bool Compiler::fgRetargetBranchesToCanonicalCallFinally(BasicBlock*      block,
 //     subsequent statements
 //   }
 //
-class FatCalliTransformer
+class IndirectCallTransformer
```

Comment:
Maybe it is time to move these helper classes out of this file. It is 25.000 lines long.

Reply:
Agree, but this sort of thing should be done on its own and not as part of a refactoring that potentially modifies behavior.